### DISCUSSION TITLE
Ktor support

### DISCUSSION THREAD

**Discussion by DONSA** [ExpediaGroup/graphql-kotlin#1677] [Q&A] [answered]
Hi,

I'm planning to build a backend GraphQL API written in Kotlin and it looks like `graphql-kotlin` is the most popular option.
I'm planning to use something like Ktor to avoid a heavy framework like Spring Boot.

I can see `graphql-kotlin` supports Ktor but the `graphql-kotlin-ktor-server` dependency in the provided [example](https://github.com/ExpediaGroup/graphql-kotlin/blob/master/examples/server/ktor-server/build.gradle.kts) is not published in the [maven repository](https://mvnrepository.com/search?q=graphql-kotlin-ktor-server).

The latest commits are still very recent so I'm wondering if this is something still being worked on.
Do you have more details about the long-term `graphql-kotlin` support for Ktor?

**Comment by dariuszkuc** [comment, marked as answer]
Hello 👋 

New Ktor GraphQL plugin is available as a preview in [v7.0.0-alpha.3](https://github.com/ExpediaGroup/graphql-kotlin/releases/tag/7.0.0-alpha.3) (released last week, see [docs](https://opensource.expediagroup.com/graphql-kotlin/docs/7.x.x/server/ktor-server/ktor-overview)). While I don't expect the API to change significantly in the final v7 release, there might be **some API** changes required to support some additional features (e.g. https://github.com/ExpediaGroup/graphql-kotlin/issues/1676).

Thanks,
Derek
